mmdet/apis/test.py

```diff
@@ -106,6 +138,18 @@ def multi_gpu_test(model, data_loader, tmpdir=None, gpu_collect=False):
             for _ in range(batch_size * world_size):
                 prog_bar.update()
 
+    # Avoid CUDA OOM, Currently only used in SOLO
+    if isinstance(results[0], DetectionResults):
+        format_results = []
+        for item in results:
```

Comment:
Why not format the results in the dataloader loop above?

Reply:
This is a time-consuming operation, I want to make the inference looks faster